src/styles/blocks/main/container.scss

```
@@ -0,0 +1,26 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(var( --column-count), $card-width);
+  column-gap: 48px;
+  row-gap: 46px;
+  justify-content: center;
+  padding: 50px 0 73px;
+}
+
+@media (min-width:488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width:768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width:1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
```

You can rewrite like this
```suggestion
.container {
  display: grid;
  grid-template-columns: repeat(var( --column-count), $card-width);
  column-gap: 48px;
  row-gap: 46px;
  justify-content: center;
  padding: 50px 0 73px;
  
  @media (min-width:488px) {
    --column-count: 2;
  }

  @media (min-width:768px) {
    --column-count: 3;
  }

  @media (min-width:1024px) {
    --column-count: 4;
  }
}
```